examples/basic/app.js

```diff
@@ -1,6 +1,30 @@
 import Vue from 'vue'
 import VueRouter from 'vue-router'
 
+// track number of popstate listeners
```

Comment:
I'm open to better ways of testing the behavior. This one directly tracks the number of popstate listeners, but we could also try to identify a specific bug that occurs due to multiple popstate events and then change the test to verify that that bug is fixed.